modules/timepoint_list/help/timepoint_list.md

```diff
@@ -1,15 +1,21 @@
 # Candidate Profile
 
-LORIS organizes longitudinal study data into a series of time-points for each candidate. Each time-point or candidate visit will be identified by a <b>Visit Label</b> (e.g. V01, V02) according to the testing plan or battery design of the study.
+LORIS organizes longitudinal study data into a series of timepoints (visit) for each candidate. Each timepoint will be identified by a **Visit Label** (i.e. V1, V2), according to the testing plan or battery design of the study.
 
-## Time Point: Actions
+## Timepoint List
 
-Authorized users can modify candidate records and add timepoints via the row of “Actions:” buttons above the "List of Visits (Time Points)" table.\r\nTo create a new time-point, click on “<b>Create Time Point</b>”.
-Users with adequate permissions may also edit general information about a candidate, such as updating participant status and general comments, by clicking the "<b>Edit Candidate Info</b>" button.
-To view all scans collected, click “<b>View Imaging Datasets</b>”, which will load the Imaging Browser timepoints available for the candidate.
-As users navigate through candidate records, clicking on the PSCID and DCCID displayed at the top of the screen will return the user to this Candidate Profile page.
+The data table lists all registered time-points, displaying summary status information about each. Click on any **Visit Label** to open a timepoint record and view all instruments and data collected at that candidate-timepoint.
 
-## Time-Point List table
+There are several actions you can take within a candidate's profile from the *Candidate Profile* page. This includes creating new timepoints, editing the candidate's information, and viewing imaging datasets.
 
-The data table lists all registered time-points, displaying summary status information about each.
-Click on any Visit Label to open a time-point record and view all instruments and data collected at that candidate-timepoint.
+## Create New Timepoint
+
+Click **Create New Timepoint**, which brings you to a new page, which has its own help text with instructions for creating a new timepoint.
+
+## Edit Candidate Info
+
+Click **Edit Candidate Info** to arrive at the *Candidate Parameters* page, where you can edit general information about the candidate, such as adding their consent. 
```

Comment:
Wasn't this button renamed? (cc @ridz1208 @johnsaigle I can't remember who did it..)

Reply:
Not ringing a bell for me

Reply:
sorry, should be just **Candidate Info** (see screenshot) - updating now. 

![Screenshot 2019-11-27 18 44 49](https://user-images.githubusercontent.com/37813606/69766458-0a839580-1146-11ea-8beb-167e29ecd747.png)

![Screenshot 2019-11-27 18 45 02](https://user-images.githubusercontent.com/37813606/69766459-0e171c80-1146-11ea-8ff9-78404b7e2615.png)